addons/popochiu/engine/objects/graphic_interface/templates/simple_click/simple_click_gui.gd

```diff
@@ -83,7 +83,7 @@ func _on_mouse_entered_inventory_item(inventory_item: PopochiuInventoryItem) ->
 	
 	if not I.active:
 		G.show_hover_text(inventory_item.description)
-	else:
+	elif I.active != inventory_item:
```

Comment:
Are you 100% sure you don't need a finalizing `else` block?

Reply:
Prolly not since the function returns `void`. It's mostly me that can't live with a dead execution branch :D